video/out/filter_kernels.c

```diff
@@ -303,6 +383,10 @@ const struct filter_kernel mp_filter_kernels[] = {
     {"sinc3",          3,   sinc},
     {"sinc4",          4,   sinc},
     {"sinc",           -1,  sinc},
+    {"ewa_lanczos16",  2,   ewa_lanczos,    .polar = true},
```

Comment:
The general idea is that the number is relative to performance, since ewa_lanczos64 uses 64 point samples (whereas lanczos2 only uses 2 per direction), but I guess it can really be argued either way.

One point of interest is that this reasoning is inconsistent with spline16/32/64, since that's also based on a separable kernel and hence indistinguishable from lanczos2/3/4 in terms of performance.

I think you're right, though, it should be named based on the number of taps.